webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
```

Comment:
please, entry and output are not necessary

Reply:
Why is it not necessary as in the requirement, Microverse never said it isn't necessary?

Reply:
you don't need configuration if you don't change the folder path. it is the one which causes your deployment to deploy some codes and ignore other codes